src/main/java/frc/robot/RobotContainer.java

```diff
@@ -359,7 +362,88 @@ public void autoChooserUpdate() {
                 new PositionIntake(m_elevator, m_wrist, () -> false, Position.STOW),
                 new WaitCommand(1.5),
                 new MoveElevator(m_elevator, ElevatorConstants.kAutoTop))),
-              new AutoDeposit(Position.TOP, m_elevator, m_wrist, m_intake, () -> false)));
+              new AutoDeposit(Position.TOP, true, m_elevator, m_wrist, m_intake, () -> false)));
+
+      m_autoCommand.addOption("ROUTINE 25: Grid 1 Two Piece Cube Top Engage",
+          Commands.sequence(
+              new AutoDeposit(Position.TOP, true, m_elevator, m_wrist, m_intake),
+              new PathPlannerCommand("Grid 1 Two Piece", 0, m_drive, true)
+                .alongWith(new WaitCommand(1.75).andThen(
+                  new PositionIntake(m_elevator, m_wrist, () -> false, Position.INTAKE), 
+                  new IntakeGamePiece(m_intake, () -> false, false))),
+              new PathPlannerCommand("Grid 1 Two Piece", 1, m_drive, true).deadlineWith(Commands.sequence(
+                new PositionIntake(m_elevator, m_wrist, () -> false, Position.STOW),
+                new WaitCommand(1.5),
+                new MoveElevator(m_elevator, ElevatorConstants.kAutoTop))),
```

Comment:
why is this a deadline

Reply:
because the auto deposit will move the elevator to the right position anyway. If it has finished the path, it should interrupt the other commands running in parallel and score the piece (the AutoDeposit, which is next in the sequence).
The only thing this needs to do is sequence(PathPlannerCommand, AutoDeposit) to move to the scoring location then score. However to be more fast it starts to move the elevator as it drives, but if it finishes driving it should move on to scoring.